src/content/docs/en/core-concepts/routing.mdx

```diff
@@ -251,7 +251,7 @@ These redirects follow the same rules as file-based routes. Dynamic routes are a
 
 Using SSR or a static adapter, you can also provide an object as the value, allowing you to specify the `status` code in addition to the new `destination`:
 
-```js
+```js title="astro.config.mjs" {5-8}
```

Comment:
Totally agree with not also highlighting `redirects` here, since we're focusing on the object that you can provide!  :raised_hands: